content/guides/destructuring.adoc

```diff
@@ -200,6 +220,34 @@ This vector is comprised of nested vectors that we can access directly.
 ;= "Line from ( 5 , 10 ) to ( 10 , 20 )"
 ----
 
+This can become arbitrarily complex:
+
+[source,clojure]
+----
+(def my-struct '[ 1 2 ( 3 4 ) 5 [ ( 6 7 ) [ ( 8 ) 9 ] ] 10 ])
```

Comment:
why all the extra spaces? here and below, that's not typical code.

Reply:
You mean,  '[1 2 (3 4) 5 [(6 7) [(8) 9 ]] 10])  ?

Reply:
Yes (still an extra space after the 9 though).